### ISSUE TITLE
Viewport addon is not registered automatically

### ISSUE THREAD

**Issue by vasilii-kovalev** [storybookjs/storybook#11848]
**Describe the bug**
As [documentation](https://github.com/storybookjs/storybook/tree/master/addons/essentials) says, Essentials addon includes both Backgrounds and Viewport addons. Backgrounds addon is registered automatically, Viewport requires manual registering.

**To Reproduce**
1. Register Essentials addon
2. Run Storybook - Viewport addon is not presented on UI
3. Register Viewport addon below
4. Re-run Storybook - Viewport addon is presented on UI

**Expected behavior**
Viewport addon is registered automatically as well as Backgrounds one.

**Screenshots**
Warning about already registered addon skipping:
![image](https://user-images.githubusercontent.com/10310491/89728943-cae1ad00-da39-11ea-850e-d71b11a6484a.png)

If registered manually, it works correctly:
![image](https://user-images.githubusercontent.com/10310491/89728964-f6fd2e00-da39-11ea-881d-5069a1ed3f88.png)

Hoping that the addon is registered automatically:
![image](https://user-images.githubusercontent.com/10310491/89728989-3166cb00-da3a-11ea-8a45-318872447f36.png)

It isn't:
![image](https://user-images.githubusercontent.com/10310491/89728999-45123180-da3a-11ea-9c59-5502d3504612.png)

**Code snippets**
Not working config in _main.ts_:
```ts
const config: StorybookConfig = {
  ...
  addons: [
    '@storybook/addon-links',
    '@storybook/addon-essentials',
    '@storybook/preset-create-react-app',
    '@storybook/addon-a11y/register',
  ],
  ...
};
```

Working config in _main.ts_:
```ts
const config: StorybookConfig = {
  ...
  addons: [
    '@storybook/addon-links',
    '@storybook/addon-essentials',
    '@storybook/preset-create-react-app',
    '@storybook/addon-a11y/register',
    '@storybook/addon-viewport/register',
  ],
  ...
};
```

**System:**
```
System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i5-8300H CPU @ 2.30GHz
  Binaries:
    Node: 14.7.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.4 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.14.7 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 84.0.4147.105
    Edge: Spartan (44.18362.449.0)
  npmPackages:
    @storybook/addon-a11y: ^5.3.19 => 5.3.19 
    @storybook/addon-actions: ^6.0.0-rc.29 => 6.0.0-rc.29 
    @storybook/addon-essentials: ^6.0.0-rc.29 => 6.0.0-rc.29
    @storybook/addon-links: ^6.0.0-rc.29 => 6.0.0-rc.29
    @storybook/node-logger: ^6.0.0-rc.29 => 6.0.0-rc.29
    @storybook/preset-create-react-app: ^3.1.4 => 3.1.4
    @storybook/react: ^6.0.0-rc.29 => 6.0.0-rc.29
```

**Comment by shilman** [comment]
It's working properly in Storybook's monorepo (NOTE: comment out the disabling code in `main.js`): https://github.com/storybookjs/storybook/tree/next/examples/cra-ts-essentials

Do you have a public repro?

**Comment by vasilii-kovalev** [comment]
Not yet. I'll do it today.

**Comment by vasilii-kovalev** [comment]
Actually I've tried to import my project via CodeSandbox, but it throws some errors. It seems, that it is related to its ecosystem.
Here the repository: https://github.com/vasilii-kovalev/games_world
I've created a `client` branch (where all the files on the first level) for convenience.

**Comment by shilman** [comment]
Cool I'll check it out

**Comment by shilman** [comment]
Fixed in #12148 and released in https://github.com/storybookjs/storybook/releases/tag/v6.0.14. Closing this for now. Please let me know if you're still seeing this after upgrading.
